x-pack/plugins/gis/public/shared/layers/sources/wms_source.js

```diff
@@ -136,13 +136,15 @@ class WMSEditor extends  React.Component {
             onChange={(e) => this._handleServiceUrlChange(e)}
           />
         </EuiFormRow>
-        <EuiFormRow label="Layers (comma-separated list)">
+        <EuiFormRow label="Layers">
           <EuiFieldText
+            placeholder={"use comma separated list of layer names"}
```

Comment:
Can just be string value but does not really matter since this will change with internationalization

`placeholder="use comma separated list of layer names"`

Reply:
Actually, can this be help text instead of a placeholder? Placeholders should be examples of what's expected not the validation.

Reply:
:+1: on helptext